Tasks/AzureRmWebAppDeployment/kuduutility.ts

```diff
@@ -175,8 +180,13 @@ export async function runPostDeploymentScript(publishingProfile, physicalPath, s
             tl.rmRF(scriptFile.filePath, true);
         }
         try {
-            await uploadFiletoKudu(publishingProfile, physicalPath, 'delete_log_file_' + uniqueID + '.cmd', path.join(__dirname, 'postDeploymentScript', 'deleteLogFile.cmd'));
-            var commandResult = await runCommandOnKudu(publishingProfile, physicalPath, 'delete_log_file_' + uniqueID + '.cmd ' + uniqueID, 0, null);
+            await uploadFiletoKudu(publishingProfile, physicalPath, 'delete_log_file_' + uniqueID + fileExtension, path.join(__dirname, 'postDeploymentScript', 'deleteLogFile' + fileExtension));
```

Comment:
should we not check here if deleteLogFiles is true ?

Reply:
@vincentdass Can you take care of this while refactoring the code? We didn't honor the value in case of windows also.

Reply:
@dikhakha , @aydey , since we delete the same file that is currently executing, it always results in error. So, we are not honoring them. Also, having the delete_file_cmd has no impact on the app.